tests/unittests/bern.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# This file was generated by tests_generator.py
+# Generated the 09/19/2017
+
+"""Original pari/GP test file bern :
+bernfrac(0);
+bernfrac(1);
+for(k = 1, 20, print(bernfrac(k)));
+for(k = 0, 5, print(bernpol(k)));
+bernfrac(-1)
+bernreal(-1)
+bernpol(-1)
+bernvec(30)
+"""
+import unittest
+from cypari2 import Pari, PariError
+
+pari = Pari()
+
+
+class TestBern(unittest.TestCase):
+    def test_bern(self):
+        pari.bernfrac(0);
```

Comment:
what this is supposed to test? and why keeping `;`?

Reply:
It test that it doesn't crash. I keep them mainly to stick with the original GP test file. 

I keep `;` during developement as an help to associate results with the corresponding test. 
Keeping them after can has no sense. I will remove them.